signing/src/test/java/tech/pegasys/web3signer/signing/config/SignerLoaderTest.java

```diff
@@ -80,40 +77,40 @@
   void signerReturnedForValidMetadataFile() throws IOException {
     final String filename = PUBLIC_KEY1 + "." + FILE_EXTENSION;
     final Path metadataFile = createFileInConfigsDirectory(filename, PRIVATE_KEY1);
-    when(signerParser.parse(Files.readString(metadataFile, StandardCharsets.UTF_8)))
+    Mockito.when(signerParser.parse(Files.readString(metadataFile, StandardCharsets.UTF_8)))
```

Comment:
Is the removal of static imports intentional? No biggy, I just think this would make it inconsistent with other tests.

Reply:
Not intentional, I prefer static imports so will fix that up

Reply:
done